src/YRichText.h

```diff
@@ -33,6 +33,9 @@
 class YRichTextPrivate;
 
 
+/**
+ * Text formatted with simple HTML-like tags, with "links" generating events.
```

Comment:
Yes, but it is out of scope for my current plan "Add missing one line descriptions to all classes".